docs/migration/README.rst

```diff
@@ -18,6 +18,66 @@ This migration combines two migrations into one:
 Performing both migrations together reduces overall work, though you need to
 perform more steps before seeing results.
 
+Ensure code repo is visible
+---------------------------
+
+App SRE only supports fetching code without authentication.  This means that
+code must either be a public repo on Github or on gitlab.cee.redhat.com.  If
+your code repo is currently private on Github, it needs to either be made public
+or moved to Gitlab.  If your repo is already public on Github or Gitlab, then
+you're good to go.
+
+Open Sourcing a Project
+^^^^^^^^^^^^^^^^^^^^^^^
+
+If you choose to keep your project on Github, it must be open sourced.  This
+usually entails several things:
+
+#. Adding a license file to the root of your repository
+#. Adding a reference to the license at the top of each source file
+#. Reviewing the repo's commit history for sensitive information
+
+Moving a Project from Github to Gitlab
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Moving a project from Github to Gitlab should be relatively straightforward, but
+will require some workflow changes for devs.
+
+#. Create a new repo on gitlab.cee.redhat.com.  There is already an
+   `insights-platform`_ group that can be used to house repos, but other groups
+   can be used if desired.
+#. Change your local upstream to gitlab.  Example::
+
+      git remote set-url upstream git@gitlab.cee.redhat.com:insights-platform/new-repo.git
+
+You shouldn't need to worry about existing CI integrations since the goal of
+this migration is to have all CI integrated into app-interface.
+
+.. _insights-platform: https://gitlab.cee.redhat.com/insights-platform/
+
+Why?
+^^^^
+
+The technical reason private Github repositories are not supported is because
+Jenkins jobs managed by App SRE do not support making authenticated pulls to git
+repos.  While support for this can of course be added, no team has been able to
```

Comment:
I think we should remove the last sentence - `While support for this can of course be added, no team has been able to
justify the work to add this support.`

I feel like it's a given that this is a technical possibility, but I don't think we want to suggest it is something that should continue to be pushed for.  Ideally, people would adjust to the workflow we're proposing here rather than expecting to rely on new dev work that isn't planned for.